portal/factories/app.py

```diff
@@ -248,9 +249,9 @@ def configure_logging(app):  # pragma: no cover
 
     # Configure Error Emails for high level log messages, only in prod mode
     if not any((
-        app.debug,
-        app.testing,
-        not app.config.get('ERROR_SENDTO_EMAIL'),
+            app.debug,
```

Comment:
Too much indentation for my taste, was this picked up by your pep8 linter?

Reply:
no, pep8 didn't see any issue -- I agree it's a bit much indentation; I think it wants to over-indent to distinguish those lines from the ``mail_handler``